python_scripts/parameter_tuning_grid_search.py

```diff
@@ -104,10 +105,10 @@
 # ## Tuning using a grid-search
 #
 # Instead of manually writing the two `for` loops, scikit-learn provides a
-# class called `GridSearchCV` which implement the exhaustive search implemented
+# class called `GridSearchCV` which implements the exhaustive search we did
```

Comment:
I think this whole sentence can be simplified. Maybe something like this (improvement welcome!):

In the previous exercise, we used multiple `for` loops to find the best hyperparameter values over a fixed grid of parameters. `GridSearchCV` is a scikit-learn class that implements a very similar logic with less boiler-plate code.

Reply:
Now that the exercises are generated automatically from the corrections, you will need to use the things introduced in https://github.com/INRIA/scikit-learn-mooc/pull/408#issuecomment-884790196